web/src/dark.scss

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
I do not get how are you getting the dark theme. Have you missed to include the actual css file/variables?

Reply:
The project uses Yeti template from Bootswatch, which is a light theme. With the new feature, when the user selects a dark theme, I use a different template (named superhero). So everything will automatically be taken care of.